src/seedu/addressbook/AddressBook.java

```diff
@@ -949,24 +1005,27 @@ private static String encodePersonToString(String[] person) {
     }
 
     /**
-     * Returns true if person data (email, name, phone etc) can be extracted from the argument string.
-     * Format is [name] p/[phone] e/[email], phone and email positions can be swapped.
+     * Returns true if person data (email, name, phone etc) can be extracted
+     * from the argument string. Format is [name] p/[phone] e/[email], phone and
+     * email positions can be swapped.
      *
-     * @param personData person string representation
+     * @param personData
+     *            person string representation
      */
     private static boolean isPersonDataExtractableFrom(String personData) {
         final String matchAnyPersonDataPrefix = PERSON_DATA_PREFIX_PHONE + '|' + PERSON_DATA_PREFIX_EMAIL;
         final String[] splitArgs = personData.trim().split(matchAnyPersonDataPrefix);
         return splitArgs.length == 3 // 3 arguments
                 && !splitArgs[0].isEmpty() // non-empty arguments
-                && !splitArgs[1].isEmpty()
-                && !splitArgs[2].isEmpty();
+                && !splitArgs[1].isEmpty() && !splitArgs[2].isEmpty();
```

Comment:
In this case is the code more readable or less?
Should magic numbers be used or constants or enums instead?